README.md

```diff
@@ -274,18 +278,34 @@ This section describes ROS topics, services and parameters that are available on
 
     The actual position of the robot on the terrain. It may be useful for validating performance of a global localization solution.
 
+* **`probe_deployment_unit/probes_dropped`** ([std_msgs/UInt8])
+
+    The actual probes number dropped. 
+
 ### Services
 
 * **`core2/reset_odometry`** ([std_srvs/Trigger])
 
-    Resets the pose published on the `wheel_pose` topic
+    Resets the pose published on the `wheel_pose` topic.
+
+* **`probe_deployment_unit/home`** ([std_srvs/Trigger])
+
+    Removes probes dropped in the simulation world.
```

Comment:
Suggested change:
Resets the PDU. In the simulation, it removes dropped probe models and resets the counter.
**Warning:** Don't use this on the real robot during the competition.